app/views/comments/_form.html.erb

```diff
@@ -66,38 +66,55 @@
 
   <%= javascript_include_tag "dragdrop" %>
   <script>
-    $('#comment-form').bind('ajax:beforeSend', function(event){      
+    $('#comment-form').bind('ajax:beforeSend', function(event){
       $("#text-input").prop('disabled',true)
-      $("#comment-form .btn-primary").button('loading',true);      
-    });
-    <% emoji_names = emoji_names_list %>
-    <% emojis = emoji_names[:emojis] %>
-    <% image_map = emoji_names[:image_map] %>
-    var image_map = <%= raw image_map.to_json %>
-    horsey(document.querySelector('textarea#text-input'), {
-        suggestions: (<%= raw emojis.to_json %>),
-        anchor: ':',
-        render: function (li, suggestion) {
-            var image = image_map[suggestion.text];
-            li.innerHTML = "<b>"+image+"</b><span>"+suggestion.text+"</span>";
-        },
-        limit: 6
+      $("#comment-form .btn-primary").button('loading',true);
     });
 
-    $('#comment-form').bind('ajax:success', function(e, data, status, xhr){ 
-      notyNotification('mint', 3000, 'success', 'topRight', 'Comment Added!');      
-      $('#text-input').prop('disabled',false);      
-      $('#text-input').val('');      
-      $('#comments-container').append(xhr.responseText);      
+    var emojis = <%== atwho_emojis %>
+    var tagnames = <%== tag_names_list %>
+
+    var at_config = {
+      at: "@",
+      callbacks: {
+        remoteFilter: function(query, callback) {
+          $.getJSON("/api/srch/profiles?srchString=" + query, {}, function(data) {
+            if (data.hasOwnProperty('items') && data.items.length > 0) {
+              callback(data.items.map(function(i) { return i.docTitle }));
+            }
+           });
+          }
+        }
+      },
+      hashtags_config = {
+        at: "#",
+        data: tagnames,
```

Comment:
OK, so here, I think you can use this syntax - it looks like it requires a POST request:

```js
$.post('/tag/suggested/' + query, function(response) {
  // do something with response, which should be an array
});
```